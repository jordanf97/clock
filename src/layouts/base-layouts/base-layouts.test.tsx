import { render } from "@testing-library/react-native";
import React from "react";
import {
  CentredKeyboardAvoidingView,
  CentredLayout,
  CentredView,
  FlexLayout,
  FlexView,
} from "..";

it("renders the FlexLayout correctly", () => {
  const tree = render(<FlexLayout />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the FlexView correctly", () => {
  const tree = render(<FlexView />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the CentredView correctly", () => {
  const tree = render(<CentredView />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the CentredLayout correctly", () => {
  const tree = render(<CentredLayout />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the CentredKeyboardAvoidingView correctly", () => {
  const tree = render(<CentredKeyboardAvoidingView />).toJSON();
  expect(tree).toMatchSnapshot();
});
