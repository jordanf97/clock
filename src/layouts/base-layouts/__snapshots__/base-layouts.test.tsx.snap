// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CentredKeyboardAvoidingView correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
/>
`;

exports[`renders the CentredLayout correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
/>
`;

exports[`renders the CentredView correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
/>
`;

exports[`renders the FlexLayout correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
/>
`;

exports[`renders the FlexView correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
/>
`;
